charm/bundle.yaml

```diff
@@ -19,8 +19,14 @@ applications:
       nagios_context: prod-event-bus-ua
       nagios_servicegroups: webops-prompt-critical,prod-event-bus-ua
     constraints: mem=4G cores=2
-  easyrsa:
-    charm: cs:~containers/easyrsa
+  vault:
+    charm: cs:vault
+    num_units: 1
+    options:
+      auto-generate-root-ca-cert: true
+      totally-unsecure-auto-unlock: true
```

Comment:
Is this intentional?

Reply:
It is. This bundle's only an example of charm usage; setting this makes it a lot easier to demo and test with.

If you don't set it, there are extra steps you have to do to initialize and unseal vault.

Reply:
Added a disclaimer comment.